webclient/.bowerrc

```diff
@@ -0,0 +1,4 @@
+{
+  "directory": "bower_components",
```

Comment:
We don't put power stuff in bower_components. It should be components, and we are already doing so, so it's probably in the bower.json